CMakeLists.txt

```diff
@@ -10,10 +10,10 @@
 # file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
 
 # We require at least CMake V3.0.2
-cmake_minimum_required(VERSION 3.0.2 FATAL_ERROR)
+cmake_minimum_required(VERSION 2.8.12 FATAL_ERROR)
```

Comment:
Why downgrading cmake from 3.x to 2.x - don't we want to push users to newer versions for better feature support?